index.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+var bodyParser = require('body-parser');
+var express = require('express');
+var GitHubServer = require('./lib/GitHubServer');
```

Comment:
We order our dependencies so that local files are listed after third-party packages, so move this to be with RegexTools below (and put a blank line in between the third-party and local files).